core-v1/src/test/scala/JustifiedSelectSpec.scala

```diff
@@ -0,0 +1,24 @@
+package com.rallyhealth.vapors.v1
+
+import data.Justified
+import example.NestedSelectable
+
+import cats.data.NonEmptyList
+import munit.FunSuite
+
+class JustifiedSelectSpec extends FunSuite {
+
+  import dsl.simple.justified._
+
+  test("select value") {
+    val root = NestedSelectable("empty").const
+    val expr = root.get(_.select(_.value))
+    val expected = Justified.byInference(
+      "select", // TODO: Put lens description here?
+      root.value.value.value,
+      NonEmptyList.of(root.value),
+    )
+    val obtained = expr.run()
+    assertEquals(obtained, expected)
+  }
+}
```

Comment:
I should add more tests here

Reply:
It looks I need to figure out how to handle the `.get(_.select(_.someOptionalOrSeqValue))` case better. In the old DSL, you would have to use `.getFoldable`, but that shouldn't be necessary anymore.